fusor-ember-cli/bower.json

```diff
@@ -1,31 +1,26 @@
 {
   "name": "fusor-ember-cli",
   "dependencies": {
-    "ember": "1.13.10",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-data": "1.13.14",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
-    "ember-qunit": "0.4.9",
-    "ember-qunit-notifications": "0.0.7",
-    "ember-resolver": "~0.1.18",
-    "jquery": "~1.11.3",
+    "ember": "~2.4.3",
```

Comment:
looks like 2.8 is released now if you feel like bumping up.

https://guides.emberjs.com/v2.8.0/